include/menu.h

```diff
@@ -47,37 +45,39 @@ struct KQuestItem
     {
     }
 
-    KQuestItem(const string& inKey, const string& inText)
+    KQuestItem(const std::string& inKey, const std::string& inText)
         : key(inKey)
         , text(inText)
     {
     }
 
-    string key;  /*!< The identifying title */
-    string text; /*!< The actual info */
+    /*! The identifying title */
+    std::string key;
+    /*! The actual info */
+    std::string text;
 };
 
 class KMenu
 {
   public:
     KMenu();
 
-    void add_questinfo(const string& key, const string& text);
+    void add_questinfo(const std::string& key, const std::string& text);
     void draw_mainmenu(int);
     void draw_playerstat(Raster* where, int player_index_in_party, int dx, int dy);
     bool give_xp(int, int, int);
-    void menu(void);
-    void revert_equipstats(void);
-    void spec_items(void);
-    void update_equipstats(void);
+    void menu();
+    void revert_equipstats();
+    void spec_items();
+    void update_equipstats();
 
   private:
     bool check_xp(int, int);
     void clear_quests();
     void level_up(int);
-    void display_quest_window(void);
+    void display_quest_window();
     void status_screen(size_t);
-    vector<KQuestItem> quest_list;
+    std::vector<KQuestItem> quest_list;
```

Comment:
Good grief, 22 / 83 files viewed, I didn't know there even was that number of files in KQ